<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnBrowse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Chọn File</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtOriginal">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>40</y>
      <width>781</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSummary">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tóm tắt</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtSummary">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>350</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu bản tóm tắt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFilepath">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chọn phương pháp</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBMethod">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
